doc/releases/changelog-dev.md

```diff
@@ -527,6 +532,10 @@
   Users should now validate these properties manually.
   [(#4773)](https://github.com/PennyLaneAI/pennylane/pull/4773)
 
+* With the aformentioned improvement to `ClassicalShadow.entropy` the keyword `atol`
+  becomes obsolete and will be removed in the next release.
+  [(#4959)](https://github.com/PennyLaneAI/pennylane/pull/4959)
```

Comment:
Given the length of the changelog, I don't think we should reference a sentence hundreds of lines away.  

```suggestion
* With an algorithmic improvement to `ClassicalShadow.entropy`, the keyword `atol`
  becomes obsolete and will be removed in v0.35.
  [(#4959)](https://github.com/PennyLaneAI/pennylane/pull/4959)
```